### PR TITLE
Installation methods: add glossary references

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Installation methods: add glossary references

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/blueprint/selectors.markdown [hunk 1]
```diff
@@ -271,7 +271,7 @@ assist_pipeline:
 ## Backup location selector
 
 This can only be used on an installation with a Supervisor (Operating System or
-Supervised). For installations of type Home Assistant Core or Container, an error
+Supervised). For installations of type {% term "Home Assistant Core" %} or Container, an error
```
Comment (c0ffeeca7) [inline source/_docs/blueprint/selectors.markdown]:
```suggestion
Supervised). For installations of type {% term "Home Assistant Core" %} or {% term "Home Assistant Container" %}, an error
```

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

File: source/_docs/configuration.markdown [hunk 2]
```diff
@@ -20,11 +20,11 @@ The path to your configuration directory can be found in the Home Assistant {% t
...
 ![Screenshot showing the top of the system information panel](/images/screenshots/System_information.png)
 
-_If you use Home Assistant Container, you can find `configuration.yaml` in the config folder that you mounted in your container._
+_If you use {% term "Home Assistant Container" %}, you can find `configuration.yaml` in the config folder that you mounted in your container._
 
-_If you use Home Assistant Operating System, you can find `configuration.yaml` in the `/config` folder of the installation._
+_If you use {% term "Home Assistant Operating System" %}, you can find `configuration.yaml` in the `/config` folder of the installation._
 
-_If you use Home Assistant Core, you can find `configuration.yaml` in the config folder passed to the `hass` command (default is `~/.homeassistant`)._
+_If you use {% term "Home Assistant Core" %} , you can find `configuration.yaml` in the config folder passed to the `hass` command (default is `~/.homeassistant`)._
```
Comment (MartinHjelmare) [inline source/_docs/configuration.markdown:27]: Side note: I noticed that "heart" in the core term is misspelled "hart".

Comment (c0ffeeca7) [reply source/_docs/configuration.markdown:27]: Thank you for the catch! https://github.com/home-assistant/home-assistant.io/pull/32288

**Pull request merged**